apps/workflowengine/lib/Manager.php

```diff
@@ -759,6 +767,7 @@ protected function getBuildInChecks(): array {
 				$this->container->query(FileName::class),
 				$this->container->query(FileSize::class),
 				$this->container->query(FileSystemTags::class),
+				$this->container->query(MfaVerified::class),
```

Comment:
```suggestion
				$this->container->get(MfaVerified::class),
```
would be the non-depracated method to use